CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.0.4
+- Add ignore dots for gmail address restriction for instance settings resource. (#293)
```

Comment:
:upside_down_face: How about...
```suggestion
- Add `IgnoreDotsForGmailAddresses` field to `InstanceRestrictions` and `instancesettings.UpdateRestrictionsParams` (#293).
```

Feel free to leave as is if you don't like the suggestion. :smile: